packages/nvidia-k8s-device-plugin/0001-Add-CDI-specs-for-the-all-device.patch

```diff
@@ -0,0 +1,45 @@
+From 1ac450cfc861200a77685dd0ba77394963217502 Mon Sep 17 00:00:00 2001
```

Comment:
Adding this for the lack of a better place:

nit: you could use the description of your patch in your commit message